tests/pipelines/test_pipeline.py

```diff
@@ -313,3 +314,24 @@ def assert_result_correct(df):
         test_df2, predictions, target_col, worst_k=2
     )
     assert_result_correct(result_df2)
+
+
+@pytest.mark.usefixtures("enter_pipeline_example_directory")
+def test_print_cached_steps_and_running_steps(capsys):
+    pipeline = Pipeline(profile="local")
+    pipeline.clean()
+    pipeline.run()
+    captured = capsys.readouterr()
+    output_info = captured.out
+    run_step_pattern = "Running step {step}..."
+    for step in _STEP_NAMES:
+        # Check for printed message when every step is actually executed
+        assert re.search(run_step_pattern.format(step=step), output_info) is not None
+
+    pipeline.run()  # cached
+    captured = capsys.readouterr()
+    output_info = captured.err
```

Comment:
I don't know why the cached info is in `stderr` while the "Running..." is in `stdout` (L325^). Both are printed with `_logger.info()`.

Reply:
The `Running...` is printed in a subprocess whose output logs are piped to the parent run's stderr